### ISSUE TITLE
AdaCos: Adaptively Scaling Cosine Logits for Effectively Learning Deep Face Representations

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/phd-bib#21]
## [AdaCos: Adaptively Scaling Cosine Logits for Effectively Learning Deep Face Representations](https://arxiv.org/abs/1905.00292)
*The cosine-based softmax losses and their variants achieve great success in deep learning based face recognition. However, hyperparameter settings in these losses have significant influences on the optimization path as well as the final recognition performance. Manually tuning those hyperparameters heavily relies on user experience and requires many training tricks. In this paper, we investigate in depth the effects of two important hyperparameters of cosine-based softmax losses, the scale parameter and angular margin parameter, by analyzing how they modulate the predicted classification probability. Based on these analysis, we propose a novel cosine-based softmax loss, AdaCos, which is hyperparameter-free and leverages an adaptive scale parameter to automatically strengthen the training supervisions during the training process. We apply the proposed AdaCos loss to large-scale face verification and identification datasets, including LFW, MegaFace, and IJB-C 1:1 Verification. Our results show that training deep neural networks with the AdaCos loss is stable and able to achieve high face recognition accuracy. Our method outperforms state-of-the-art softmax losses on all the three datasets.*

### Cite this paper
```bibtex
@inproceedings{zhang2019adacos,
  title={Adacos: Adaptively scaling cosine logits for effectively learning deep face representations},
  author={Zhang, Xiao and Zhao, Rui and Qiao, Yu and Wang, Xiaogang and Li, Hongsheng},
  booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
  pages={10823--10832},
  year={2019}
}
```

**Comment by juanmc2005** [comment]
## Summary

Interesting analysis of the effect of margin and scaling hyper-parameters in cosine losses.
They proposed a new loss function to adapt the scaling hyper-parameter and removing the margin, as their effects overlap.

Even if a non-adaptive cosine loss function is used, they give a good starting point for scaling and perform a detailed analysis on how changing these parameters affects the model.
